insights/parsers/cpu_vulns.py

```diff
@@ -0,0 +1,33 @@
+"""
+Parsers for cpu vulnerabilities file output
+===========================================
+
+This module provides the following parser:
+
+CpuVulns - ``/sys/devices/system/cpu/vulnerabilities/*``
+"""
+
+from __future__ import division
```

Comment:
Just curious, is there any reason for this import statement? If so please add comment about it.

Reply:
It was for python 2 on some digital computation, it doesn't impact much in current parser. I can remove it. Thanks for your comment.